src/etc/inc/shaper.inc

```diff
@@ -4762,9 +4762,11 @@ class dnqueue_class extends dummynet_class {
 			'Mask',
 			$mask['type'],
 			array('none' => gettext('None'), 'srcaddress' => gettext('Source addresses'), 'dstaddress' => gettext('Destination addresses'))
-		))->setHelp('If "source" or "destination" slots is chosen a dynamic pipe with the bandwidth, delay, packet loss ' .
+		))->setHelp('If "source" or "destination" slots is chosen a dynamic queue with the bandwidth, delay, packet loss ' .
 					'and queue size given above will be created for each source/destination IP address encountered, respectively. ' .
-					'This makes it possible to easily specify bandwidth limits per host.');
+ 					'This makes it possible to easily specify bandwidth limits per ' .
+ 					'host or subnet, usually capped by the parents limiters bandwidth ' .
```

Comment:
"usually capped by the parents limiters bandwidth" isn't grammatically correct. It should either be:

* "usually capped by the bandwidth of the parent limiter"
or
* "usually capped by the parent limiter's bandwidth"

As we try to avoid using awkward possessive forms ([ref](https://docs.netgate.com/pfsense/en/latest/references/style-guide.html)), the first choice is best.

Reply:
Thanks for pointing that out to my non-native tongue. Fixed. And read the style guide now.